### ISSUE TITLE
arangoimp csv ignores last line if not followed by \n

### ISSUE THREAD

**Issue by baslr** [arangodb/arangodb#2342]
ArangoDB v3.1.12
csv file that imports only the first entry
```
_key,xx,blarg
"499999",1,"blub"
"499998",1,"blub"
```
csv file that imports both entries
```
_key,xx,blarg
"499999",1,"blub"
"499998",1,"blub"

```

Note that vim will add independently a newline. Check with cat or xxd for a newline at the end of the file.

arangoimp cmd:
```
./build/bin/arangoimp --collection testcsv --create-collection true \
 --separator , --type csv test.csv
```

**Comment by jsteemann** [comment]
Fixed in 3.1 and devel.
By the way, I prefer using vim with `set noeol` for saving and `hexdump -C` for checking.

**Comment by dothebart** [comment]
ArangoDB 3.1.13 is available for download.
